TestFramework/ExtensionMethods.cs

```diff
@@ -534,5 +534,30 @@ public static void SwitchToIframe(IWebDriver driver, By by)
         {
             driver.SwitchTo().Frame(driver.FindElement(by));
         }
+
+        public static void WaitForTextPresent(IWebDriver driver, string text, int? timeInSec = null)
+        {
+            bool isPresent = false;
+            timeInSec = timeInSec == null ? 30 : timeInSec.Value;
+            int count = 1;
+            while (!isPresent)
+            {
+                try
+                {
+                    WebDriverWait wait = new WebDriverWait(driver, TimeSpan.FromSeconds(1));
```

Comment:
I see a wait has been declared, however it has not never been used in the method. If that's the case, we can remove line 547